Fix minimum visible blocks tooltip showing even though minVisibleBlocks is not defined (#2360)
vivid-planet/comet: .changeset/cold-bags-walk.md

---
"@comet/blocks-admin": patch
---

> createListBlock: Don't show the minimum required blocks tooltip when the option isn't used

Reviewer: ```suggestion
createListBlock: Don't show the minimum visible blocks tooltip when the option isn't used
```

> This fixes a bug where a tooltip for the minimum amount of required blocks of a `LinkBlock` was shown even if the option was not defined in the options.

Reviewer: ```suggestion
createListBlock: Prevent tooltip for minimum required blocks showing even when the `minVisibleBlocks` isn't used.
```
